### PR TITLE
fixes url loading sync issue for actions

### PR DESCRIPTION
**Pull request by nesalang**
removed hard coded image icon placement that causes misalignment of the icons in the buttons & fixed sync issues with concurrent queue.

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm [hunk 1]
```diff
@@ -216,8 +216,10 @@ - (void)addTasksToConcurrentQueue:(std::vector<std::shared_ptr<BaseCardElement>>
                     img->SetImageSize(imgSetElem->GetImageSize());
 
                     if([rendererRegistration isElementRendererOverriden:(ACRCardElementType) CardElementType::Image] == NO){
-                        /// tag a base card element with unique key
-                        [self processImageConcurrently:img];
+                    NSString *urlStr = [NSString stringWithCString:img->GetUrl().c_str()
+                                                          encoding:[NSString defaultCStringEncoding]];
+                    NSString *key = [ACRView generateKeyForElement:img];
+                    [self loadImage:urlStr key:key];
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: Indent code.

Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm]: sorry

---
In reply to: [197509077](https://github.com/Microsoft/AdaptiveCards/pull/1641#discussion_r197509077) [](ancestors = 197509077)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.h [hunk 2]
```diff
@@ -11,7 +11,8 @@
 @interface ACOHostConfig:NSObject
 
 @property NSArray<NSString *> *fontFamilyNames;
-
+@property BOOL areAllActionsHaveIcons;
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.h]: >areAllActionsHaveIcons [](start = 15, length = 22)

Grammatically, this should be `doAllActionsHaveIcons`, or preferably just `allActionsHaveIcons`.

Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.h]: thanks :-)

---
In reply to: [197509368](https://github.com/Microsoft/AdaptiveCards/pull/1641#discussion_r197509368) [](ancestors = 197509368)

**Pull request merged**